lib/binaries/binary.ts

```diff
@@ -21,7 +21,6 @@ export interface BinaryMap<T extends Binary> { [id: string]: T; }
 export class Binary {
   static os: OS[];                // the operating systems, the binary can run on
   static id: string;              // the binaries key identifier
-  static isDefault: boolean;      // to download by default
   static versionDefault: string;  // a static default version variable
   static shortName: string[];     // the names used for a binary download
```

Comment:
If `shortName` is gone remove it here too.  Please do a `grep` too do double check